docs/install/rancher.md

```diff
@@ -1,248 +1,159 @@
 ---
-title: "Rancher"
-linkTitle: "Rancher"
-weight: 30
+title: "Rancher Kubernetes Engine (RKE)"
+linkTitle: "Rancher Kubernetes Engine (RKE)"
+weight: 1
```

Comment:
I can see multiple weight:1 on the installation pages, that means that there is no order. 

The lower the number the higher on the menu. I would recommend to use multiples of 10 or 100, so we can put weights in the middle if new pages are created without having to alter all other documents. 

So we could do

```
kubernetes.md -> weight: 10
openshift.md -> weight: 20
rancher.md -> weight: 30
amazon-web-services-eks.md -> weight: 40
google-kubernetes-engine-gke.md -> weight: 50
Anthos.md -> weight: 60
digitalocean-kubernetes.md -> weight: 70
microsoft-azure-aks.md -> weight: 80
airgapped-clusters.md -> weight: 90
```

Right now there are some with real weights and some with 1.

Reply:
@Arau @cannischan as discussed earlier, lets keep the weighting to 1 for now (which seems to default to alphabetical order) for the different kubernetes distributions and then put a larger weight for air gapped and declarative guide.